crates/services/consensus_module/poa/src/lib.rs

```diff
@@ -3,16 +3,20 @@
 #![deny(unused_crate_dependencies)]
 #![deny(unused_must_use)]
 #![deny(warnings)]
+#![allow(clippy::blocks_in_conditions)] // False positives with tracing macros
 
 mod deadline_clock;
 mod sync;
 
 #[cfg(test)]
 mod service_test;
 
+use aws_config as _; // todo
```

Comment:
`todo` what?

Reply:
Figure out if we need the dependency at all, or behind a feature flag. Turns out tt's not needed at all. Removed in 3c0f4ccc9.